baybe/recommenders/pure/bayesian/sequential_greedy.py

```diff
@@ -183,9 +186,10 @@ def _recommend_hybrid(
 
         # Actual call of the BoTorch optimization routine
         try:
+            # TODO Roya convert
```

Comment:
whats left to do here?

Reply:
@rjavadi please clarify

Reply:
@rjavadi I saw this was fixed. In the future, please can you comment on the individual threads with a quick 'changed' or 'fixed' or similar before resoling them? Otherwise its almost impossible for us to follow the status of requested changes and clarifications. Merging now